doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -62,6 +62,7 @@ Please make sure to set your preferred storage location ***before*** migration.
   removed. Instead, please pass the arguments as an associative array (eg.`getNavigation($args)`.For details, please see [#12310](https://github.com/pimcore/pimcore/issues/12310)
 - [CoreBundle] Please update CoreBundle config resource path from `@PimcoreCoreBundle/Resources/config/...` to `@PimcoreCoreBundle/config/..` in your project configurations.
 - [Email] Bumped `league/html-to-markdown` to ^5.1.
+- [Logger] Bumped `monolog/monolog` to [^3.0](https://github.com/Seldaek/monolog/releases/tag/3.0.0) and `symfony/monolog-bundle` to [^3.8](https://github.com/symfony/monolog-bundle/releases/tag/v3.8.0) (which adds support for monolog v3). Please adapt your custom implementation accordingly eg. log records are now `LogRecord` Objects instead of array.
```

Comment:
```suggestion
- [Logger] Bumped `monolog/monolog` to [^3.2](https://github.com/Seldaek/monolog/blob/main/UPGRADE.md#300) and `symfony/monolog-bundle` to [^3.8](https://github.com/symfony/monolog-bundle/blob/master/CHANGELOG.md#380-2022-05-10) (which adds support for monolog v3). Please adapt your custom implementation accordingly eg. log records are now `LogRecord` Objects instead of array.
```